common.gypi

```diff
@@ -47,18 +47,22 @@
         'v8_postmortem_support%': 'false',
         'OBJ_DIR': '<(PRODUCT_DIR)/obj',
         'V8_BASE': '<(PRODUCT_DIR)/lib/v8_libbase.lib',
+        'lib_prefix': '',
+        'static_lib_suffix': '.lib',
       }, {
         'os_posix': 1,
         'v8_postmortem_support%': 'true',
+        'lib_prefix': 'lib',
+        'static_lib_suffix': '.a',
```

Comment:
These are built-in as `STATIC_LIB_PREFIX` and `STATIC_LIB_SUFFIX`, i.e., you could write the line below as:
```python
'OPENSSL_PRODUCT': '<(STATIC_LIB_PREFIX)crypto<(STATIC_LIB_SUFFIX)',
```

Reply:
Oh that is great, I'll change that.

Reply:
Can you remove the variables now? :-)